scripts/ci_build.sh

```diff
@@ -19,6 +19,8 @@
 
 set -e
 
+time ./scripts/integration_test.sh || exit 1
```

Comment:
Should we keep that wrapped up with `time` command?

Reply:
Oh, now I think about it, it probably interfers with the exit 1 code. Will remove on the other PR